_posts/2021-06-06-announcing_trino_summit.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Announcing Trino Summit"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Greetings Trino Nation,
+
+As mentioned in [last year’s Presto Summit blog](https://blog.starburst.io/presto-summit-sf-postponed-to-2021), 
+“It’s not goodbye, it’s see you later!” We may not have guessed at the time that
+“later” was a year and a half, but here we are! Get ready for this year’s hybrid
```

Comment:
Would suggest adding a line break so "Get ready for this year's hybrid Trino Summit event" is the bombastic start of its own paragraph and not buried in the middle of one.

Reply:
bombastic...awesome word